quodlibet/quodlibet/formats/_audio.py

```diff
@@ -532,26 +533,35 @@ def list(self, key):
             v = self.get(key)
             return [] if v is None else v.split("\n")
 
-    def list_separate(self, key, connector=" - "):
-        """Similar to list, but will return a list of all combinations
-        for tied tags instead of one comma separated string.
-
-        In case of tied tags the result will be unicode, otherwise
-        it returns the same as list()
+    def list_sort(self, key):
+        """Like list but return display,sort pairs when appropriate
+        and work on all tags
         """
+        display = decode_value(key, self.__call__(key))
```

Comment:
OK, fixed (in two places).

peter


On 01/22/2016 10:00 AM, Christoph Reiter wrote:
> In quodlibet/quodlibet/formats/_audio.py
> <https://github.com/quodlibet/quodlibet/pull/1796#discussion_r50568908>:
> 
>>          """
>> +        display = decode_value(key, self.__call__(key))
> 
> note that |self.__call__()| is the same as |self()|
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quodlibet/quodlibet/pull/1796/files#r50568908>.
>